log/mod.ts

```diff
@@ -23,22 +23,23 @@ export interface LogConfig {
 const DEFAULT_LEVEL = "INFO";
 const DEFAULT_NAME = "";
 const DEFAULT_CONFIG: LogConfig = {
-  handlers: {},
+  handlers: {
+    "default": new ConsoleHandler(DEFAULT_LEVEL)
+  },
 
   loggers: {
+    "default": {
+      level: DEFAULT_LEVEL,
+      handlers: ["default"]
+    },
     "": {
```

Comment:
This won't be needed any more

Reply:
Mind expanding on what you mean here? Suggestions of a cleaner way to do this would be nice if you think this change isn't adequate.

The whole reason this file changed at all is because `defaultHandler` and `defaultLogger` can't be changed for test purposes, so we need a way to _set_ the defaults like any other logger/handler. There are places we reference `defaultLogger` by ref, which means we can't do that.

Reply:
That's fine and I agree there should be a way to configure default logger. I meant that `""` logger is not needed anymore and can be removed.